### PR TITLE
Documentation input+regions options in TimezoneType

### PR DESCRIPTION
**Pull request by ro0NL**
See https://github.com/symfony/symfony/pull/23648

### CODE REVIEW

File: reference/forms/types/timezone.rst [hunk 1]
```diff
@@ -45,6 +48,27 @@ manually, but then you should just use the ``ChoiceType`` directly.
...
+The format of the *input* data - i.e. the format that the timezone is stored
+on your underlying object. Valid values are:
+
+* ``string`` (e.g. ``America/New_York``)
+* ``datetimezone`` (a ``DateTimeZone`` object)
+
+regions
+~~~~~~~
+
+**type**: ``int`` **default**: ``DateTimeZone::ALL``
```
Comment (HeahDude) [inline reference/forms/types/timezone.rst]: `\DateTimeZone::ALL`

**Comment by ro0NL** [comment]
ping @HeahDude

**Comment by ro0NL** [comment]
Guys.. this should be merged really :) cc @javiereguiluz

**Comment by javiereguiluz** [comment]
I'm sorry it wasn't merged before. The problem is that currently doc mergers have more work than they can handle. It will take some time to merge this. Please bear with us in these stressful moments before the Symfony 4 release. 🙏

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you Roland.

**Pull request merged**